blockchain/validators/src/metadata/avr/config.rs

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Facilities for creating an AVR verifier object from a configuration file.
+
+use crate::{
+    error::ParseError,
+    metadata::avr::{AvrValidationRecord, AvrValidator},
+    ValidationError,
+};
+use mc_attest_core::serial;
+use mc_attest_verifier::Verifier;
+use mc_blockchain_types::{BlockIndex, VerificationReport};
+use mc_common::ResponderId;
+use mc_crypto_keys::Ed25519Public;
+use serde::{Deserialize, Serialize};
+use std::{collections::HashMap, ffi::OsStr, fs, path::Path};
+use TryFrom;
+
+const IAS_ANCHORS: &[&str] = &[include_str!(
+    "../../../../../attest/verifier/data/AttestationReportSigningCACert.pem"
+)];
```

Comment:
> @eranrund commented
> I suspect this is problematic, since it forces production certificates even for networks that use the dev certificates. This should probably use the code in the attest verifier crate that selects the right certs based on the build environment variables.

Reply:
Yeah it might be. And I'm not entirely sure yet what the right solution is?

It seems to me that if we're trying for Avr Verification - we do want to try to ensure that the AVR was generated from intel's chain of trust? Otherwise people could be feeding data that mimics AVRs from Intel signed enclaves with a false "block signing key" that matches that in the fake AVR.

For these tests we can switch to test data so production certificates don't have to be on chain, and potentially disable this step in test deploys?